src/components/manage/Blocks/Image/Edit.jsx

```diff
@@ -265,31 +265,62 @@ class Edit extends Component {
         )}
       >
         {data.url ? (
-          <img
-            className={cx({
-              'full-width': data.align === 'full',
-              large: data.size === 'l',
-              medium: data.size === 'm',
-              small: data.size === 's',
-            })}
-            src={
-              isInternalURL(data.url)
-                ? // Backwards compat in the case that the block is storing the full server URL
-                  (() => {
-                    if (data.size === 'l')
+          <figure
+            className={cx(
+              'figure',
+              {
+                center: !Boolean(data.align),
+              },
+              data.align,
+              {
+                'full-width': data.align === 'full',
+                large: data.size === 'l',
+                medium: data.size === 'm' || !data.size,
+                small: data.size === 's',
+              },
+            )}
+          >
+            <img
+              src={
+                isInternalURL(data.url)
+                  ? // Backwards compat in the case that the block is storing the full server URL
+                    (() => {
+                      if (data.size === 'l')
+                        return `${flattenToAppURL(data.url)}/@@images/image`;
+                      if (data.size === 'm')
+                        return `${flattenToAppURL(
+                          data.url,
+                        )}/@@images/image/preview`;
+                      if (data.size === 's')
+                        return `${flattenToAppURL(
+                          data.url,
+                        )}/@@images/image/mini`;
                       return `${flattenToAppURL(data.url)}/@@images/image`;
-                    if (data.size === 'm')
-                      return `${flattenToAppURL(
-                        data.url,
-                      )}/@@images/image/preview`;
-                    if (data.size === 's')
-                      return `${flattenToAppURL(data.url)}/@@images/image/mini`;
-                    return `${flattenToAppURL(data.url)}/@@images/image`;
-                  })()
-                : data.url
-            }
-            alt={data.alt || ''}
-          />
+                    })()
+                  : data.url
+              }
+              alt={data.alt || ''}
+            />
+            {data?.description && (
+              <figcaption className="description">
+                {data?.description}
+              </figcaption>
+            )}
+            {data?.rights && (
+              <>
+                <figcaption
+                  className={cx('copyright', {
+                    nodescription: !data?.description,
+                  })}
+                >
+                  <em>Copyright: </em>
```

Comment:
@ThomasKindermann  you should add an i18n message instead of having it hard coded in English.
 I also wonder if you shouldn't allow the user to write any message from this copyright field.